lib/exporters/jhipster_entity_exporter.js

```diff
@@ -92,7 +93,9 @@ function exportEntities(passedConfiguration) {
   if (shouldFilterOutEntitiesBasedOnMicroservice()) {
     filterOutEntitiesByMicroservice();
   }
-  writeEntities(passedConfiguration.application.forSeveralApplications ? configuration.application.name : '');
+  if (!passedConfiguration.configuration || !passedConfiguration.configuration.skipEntityFilesGeneration) {
```

Comment:
Might as well create a variable for that `!passedConfiguration.configuration || !passedConfiguration.configuration.skipEntityFilesGeneration`, `skippedEntityFileGenerations`